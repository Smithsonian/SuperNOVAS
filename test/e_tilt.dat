
2450545.000000 TEST       S2 O0 A1:    23.439636    23.437215    -0.072932    -1.192454    -8.715705

2450545.000000 TEST       S2 O1 A1:    23.439636    23.437215    -0.072932    -1.192454    -8.715705

2450545.000000 TEST       S2 O2 A1:    23.439636    23.437215    -0.072932    -1.192454    -8.715705

2450545.000000 SUN        S0 O0 A1:    23.439636    23.437215    -0.072932    -1.192454    -8.715705

2450545.000000 SUN        S0 O1 A1:    23.439636    23.437215    -0.072932    -1.192454    -8.715705

2450545.000000 SUN        S0 O2 A1:    23.439636    23.437215    -0.072932    -1.192454    -8.715705

2451545.000000 TEST       S2 O0 A1:    23.439279    23.437677    -0.852016   -13.931977    -5.769410

2451545.000000 TEST       S2 O1 A1:    23.439279    23.437677    -0.852016   -13.931977    -5.769410

2451545.000000 TEST       S2 O2 A1:    23.439279    23.437677    -0.852016   -13.931977    -5.769410

2451545.000000 SUN        S0 O0 A1:    23.439279    23.437677    -0.852016   -13.931977    -5.769410

2451545.000000 SUN        S0 O1 A1:    23.439279    23.437677    -0.852016   -13.931977    -5.769410

2451545.000000 SUN        S0 O2 A1:    23.439279    23.437677    -0.852016   -13.931977    -5.769410

2452545.000000 TEST       S2 O0 A1:    23.438923    23.439867    -1.031818   -16.872017     3.396111

2452545.000000 TEST       S2 O1 A1:    23.438923    23.439867    -1.031818   -16.872017     3.396111

2452545.000000 TEST       S2 O2 A1:    23.438923    23.439867    -1.031818   -16.872017     3.396111

2452545.000000 SUN        S0 O0 A1:    23.438923    23.439867    -1.031818   -16.872017     3.396111

2452545.000000 SUN        S0 O1 A1:    23.438923    23.439867    -1.031818   -16.872017     3.396111

2452545.000000 SUN        S0 O2 A1:    23.438923    23.439867    -1.031818   -16.872017     3.396111
