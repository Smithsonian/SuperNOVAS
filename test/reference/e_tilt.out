  -10000.000 A0:    23.442841    23.443966     1.003857    16.415275     4.048548
       0.000 A0:    23.439279    23.437677    -0.852015   -13.931964    -5.769382
   10000.000 A0:    23.435717    23.437466     0.648421    10.602876     6.295088
   10000.000 A0:    23.435717    23.437466     0.648421    10.602876     6.295088
   10000.010 A0:    23.435717    23.437466     0.648446    10.603298     6.294377
  -10000.000 A1:    23.443       23.444        1.004       16.414        4.049
       0.000 A1:    23.439       23.438       -0.852      -13.932       -5.769
   10000.000 A1:    23.436       23.437        0.648       10.602        6.295
   10000.000 A1:    23.436       23.437        0.648       10.602        6.295
   10000.010 A1:    23.436       23.437        0.648       10.603        6.295
